### PR TITLE
First Creation of Ops-UI

### PR DESCRIPTION
**Pull request by LeoKHoward**
First stab at creating census-rm-ops-ui repo

### CODE REVIEW

File: run.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import os
+
+from flask import Flask
+
+app = Flask(__name__)
+
+
+@app.route('/')
+def hello_world():
+    return "Hello World!"
```
Comment (NickGrantX) [inline run.py]: I would prefer that it said "Hello World" using a Jinja template, given that's the tech stack we're going to be using... not writing HTML into the HTTP response directly

**Review by NickGrantX — CHANGES_REQUESTED** [review]
Needs Jinja

File: run.py (hunk 1 above)
Comment (AdamHawtin) [reply run.py]: Is that necessary? The aim of the ticket was just to stand _something_ up as a placeholder for the ops UI.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
+sudo: required
+
+services:
+  - docker
+
+language: python
+
+python:
+  - "3.6"
```
Comment (ryangrundy7) [inline .travis.yml]: Should be the same as whats in the pipfile/dockerfile. Could even go to 3.8 but I don't think any of our apps are on that yet so put it to 3.7

File: Makefile [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+	PIPENV_PYUP_API_KEY="" pipenv check ./app
+
+start:
+	pipenv run python run.py
+
+docker:
+	docker build -t eu.gcr.io/census-rm-ci/rm/census-rm-ops-ui .
+
+docker-run: docker
+	docker run --network=censusrmdockerdev_default  -p 8003:80 eu.gcr.io/census-rm-ci/rm/census-rm-ops-ui:latest
```
Comment (ryangrundy7) [inline Makefile]: Keep the port consistent with whats in the docker-compose file in docker-dev. This port is currently the ops-tool port so wouldn't run

**Review by ryangrundy7 — DISMISSED** [review]
Should create dockerignore and gitignore files as well so we don't upload anything we don't want. Looks like you've accidentally pushed some `__pycache__` which should be removed

File: run.py (hunk 1 above)
Comment (NickGrantX) [reply run.py]: I don't think it adds much effort but happy to recant if @LeoKHoward doesn't want to do it

Comment (LeoKHoward) [reply run.py:10]: Tried finding the solution but couldn't find a clear cut one so just gonna leave it for now

**Review by NickGrantX — CHANGES_REQUESTED** [review]
You need to get rid of that `__pycache__/run.cpython-37.pyc` file

**Review by NickGrantX — APPROVED** [review]
LGTM 👍

File: .travis.yml [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+python:
+  - "3.7"
+
+install:
+  - pip install pipenv
+  - pipenv install --dev --deploy
+
+script:
+  - make build
```
Comment (AdamHawtin) [inline .travis.yml]: This should be 
```shell-script
- make lint
- make docker
```
Build only does a pipenv install which is already done in the install step.

**Review by AdamHawtin — APPROVED** [review]
LGTM!

**Pull request merged**